### DISCUSSION TITLE
Give up on GitHub???

### DISCUSSION THREAD

**Discussion by tpike3** [Open-Source-Software-Neighborhood-Watch/town-square#16] [General]
I was just pointed to this [Give Up on GitHub](https://sfconservancy.org/GiveUpGitHub/). Generally from an open source perspective it raises some interesting questions. 

Generically, I will contend if something is all public and there is no benefit for the creator (i.e. there is no incentive for the hard work) is bad, but all corporate (e.g. there is no individual innovation, everything is owned by a business) is bad too. 

I know of no way to determine the right mix, but it seems like a great complex systems question though-- How doe we find the edge of chaos?

**Comment by jspeed-meyers** [comment]
The Software Freedom Conservancy is an important and interesting organization. I take this editorial and manifesto seriously.

That said, the underpinning logic of the argument isn't convincing, although I agree with Tom's sentiment that is a hard question to settle, even with ample research resources.

The key assumptions in this article are:

- Copyleft licenses like GPL are beneficial. (It's never spelled out exactly how.)
- FOSS developers should not rely on proprietary products, especially for code hosting.
- Companies should not sell to ICE or other "problematic" entities.
- GitHub co-pilot ought to credit developers or, perhaps, be re-created without using copyleft code, or perhaps be discontinued.

Those are just a few. These are big and important claims. Let's discuss at the next meeting! 

wdyt?
